docs/smoke-test.md

```
@@ -215,6 +216,81 @@ Document for each test run:
 
 ---
 
+### 11. POST_NOTIFICATIONS Denial Path (Android 13+ only)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)